package.json

```diff
@@ -35,12 +30,13 @@
         "eslint-plugin-vue": "^7.20.0",
         "expr-eval": "^2.0.2",
         "js-yaml-loader": "^1.2.2",
-        "laravel-mix": "^6.0.49",
         "moment": "^2.29.1",
         "moment-timezone": "^0.5.31",
         "resolve-url-loader": "^3.1.2",
         "sass-loader": "^12.6.0",
```

Comment:
a ticket can be created to move from `sass-loader` to `sass` , dependency swap, no code changes needed